keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/callbacks/SlackCallbackHandler.kt

```diff
@@ -0,0 +1,89 @@
+package com.netflix.spinnaker.keel.slack.callbacks
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.netflix.spinnaker.keel.notifications.NotificationType.MANUAL_JUDGMENT
+import com.netflix.spinnaker.keel.serialization.configuredObjectMapper
+import com.netflix.spinnaker.kork.web.exceptions.InvalidRequestException
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.net.URLDecoder
+
+@Component
+class SlackCallbackHandler(
+  private val manualJudgmentCallbackHandler: ManualJudgmentCallbackHandler
+) {
+  companion object {
+    val mapper: ObjectMapper = configuredObjectMapper()
+  }
+
+  private val log by lazy { LoggerFactory.getLogger(javaClass) }
+
+  fun handleCallback(request: String?) {
+    if (request == null) {
+      log.error("the request body coming from slack callback is null.")
+      //throw exception?
+      return
+    }
+
+    val payload = parseSlackPayload(request)
```

Comment:
From the [Slack Java SDK docs](https://slack.dev/java-slack-sdk/guides/interactive-components), it looks like you don't have to do all this yourself -- it's all built into the SDK.

Example:
```kotlin
app.blockAction("button-action") { req, ctx ->
  val value = req.payload.actions[0].value
  if (req.payload.responseUrl != null) {
    ctx.respond("You've sent ${value} by clicking the button!")
  }
  ctx.ack()
}
```

Reply:
done!